static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -227,7 +265,13 @@ export function ThresholdGroupRows({
   const editThresholdState = (thresholdId, key, value) => {
     if (editingThresholds[thresholdId]) {
       const updateEditing = JSON.parse(JSON.stringify(editingThresholds));
+      const currentThresholdValues = updateEditing[thresholdId];
       updateEditing[thresholdId][key] = value;
+      if (key === 'threshold_type' && value === CRASH_FREE_SESSION_RATE_STR) {
+        if (['seconds', 'minutes'].indexOf(currentThresholdValues.windowSuffix) > -1) {
+          updateEditing[thresholdId].windowSuffix = 'hours';
+        }
```

Comment:
does this force the editing window to hours if the threshold is for crash free session rates?

Reply:
Yup exactly